mlflow/utils/autologging_utils/safety.py

```diff
@@ -539,7 +541,16 @@ def call_original(*og_args, **og_kwargs):
                 else:
                     return original(*args, **kwargs)
 
-    _wrap_patch(destination, function_name, safe_patch_function)
+    new_patch = _wrap_patch(destination, function_name, safe_patch_function)
+    _store_patch(autologging_integration, new_patch)
+
+
+def revert_patches(autologging_integration):
+    if autologging_integration in _AUTOLOGGING_PATCHES:
```

Comment:
Added check for existance, because if not throws a  `None is non-iterable` exception.

Reply:
@mohamad-arabi Can we do the following to avoid the duplicate existence check?

```
for patch in _AUTOLOGGING_PATCHES.get(autologging_integration, []):
```